.github/workflows/check_node_latest.yml

```diff
@@ -0,0 +1,38 @@
+name: Check Reflex run on Node.js Latest
+
+on:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+env:
+    TELEMETRY_ENABLED: false
+
+jobs:
+    check_latest_node:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                python-version: ['3.12']
+                node-version: ['node']
+        steps:
+            - uses: actions/checkout@v4
+            - uses: ./.github/actions/setup_build_env
+              with:
+                python-version: ${{ matrix.python-version }}
+                run-poetry-install: true
+                create-venv-at-path: .venv
+            - uses: actions/setup-node@v4
+              with:
+                node-version: ${{ matrix.node-version }}
+            - run: |
+                poetry run uv pip install playwright pytest-playwright pyvirtualdisplay pillow
```

Comment:
we need to add pytest-playwright to dev dependencies, and probably remove playwright and playwright-pytest from here

```
  def test_node_version(node_version_app: AppHarness, page: Page):
E       fixture 'page' not found
>       available fixtures: anyio_backend, anyio_backend_name, anyio_backend_options, app_harness_env, benchmark, benchmark_weave, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, class_mocker, cov, doctest_namespace, event_loop, mocker, module_mocker, monkeypatch, no_cover, node_version_app, package_mocker, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, session_mocker, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory, xvfb
>       use 'pytest --fixtures [testpath]' for help on them.
```